templates/shared/forms/interactive/real-time.html

```diff
@@ -5,7 +5,7 @@
       <h2 class="p-modal__title u-sv1" id="modal-title">Run Real-time Ubuntu in production</h2>
     </header>
     <div class="js-pagination js-pagination--1">
-      <form action="/marketo/submit" method="post" id="mktoForm_%% formid %%" class="modal-form p-form p-form--stacked" onsubmit="buildCommentsForLead()">
+      <form action="/marketo/submit" method="post" id="mktoForm_%% formid %%" class="modal-form p-form p-form--stacked" onsubmit="buildCommentsForLead('mktoForm_%% formid %%')">
```

Comment:
This is a modal form and the spinner should be handled with [dynamic-forms.js](https://github.com/canonical/ubuntu.com/blob/main/static/js/src/dynamic-forms.js#L532C11-L532C12), it doesn't work on this form for some reason though. Can you investigate why?